gqcp/include/Processing/DensityMatrices/OneDM.hpp

```diff
@@ -44,6 +44,17 @@ class OneDM: public QCMatrix<_Scalar> {
      */
 
     using QCMatrix<Scalar>::QCMatrix;  // inherit base constructors
+
+    /*
+     * PUBLIC METHODS
+     */
+    
+    /**
+     *  @param T          transformation matrix for the spin unresolved 1-DM
+     * 
+     *  @return the transformed density matrix.
+     */
+    OneDM<Scalar> transform(const TransformationMatrix<double>& T) const { return this->basisTransform(); }
```

Comment:
```suggestion
    OneDM<Scalar> transformed(const TransformationMatrix<double>& T) const { return this->basisTransform(T); }
```

I guess you forgot an argument here. Also, I'd like to suggest the name `transformed` instead of `transform`, indicating that [a modification is being returned to the caller](https://swift.org/documentation/api-design-guidelines/).